en/getting_started/flight_modes.md

```diff
@@ -169,16 +173,36 @@ This is the GPS guided mode. You can fly in Position Mode if you need precision
+
+[Altitude mode](../flight_modes/altitude_fw.md) makes it easier for users to control vehicle altitude, and in particular to reach and maintain a fixed altitude. The mode does not use GPS, and will not attempt to hold the vehicle course against wind.
+
+The climb/descent rate is controlled via the pitch/elevator stick. Once centered the autopilot latches onto the current altitude and will maintain it during yaw/roll, and at any airspeed. The throttle input controls airspeed. 
+
+When all remote control inputs are centered (no roll, pitch, yaw, and ~50% throttle) the aircraft will return to straight, level flight (subject to wind) and keep its current altitude.
+
+> **Tip** *Altitude mode* is the safest non GPS guided mode appropriate for beginners learning how to fly. It is just like [Manual](#manual_fw) mode but additionally stabilizes the vehicle altitude when the pitch stick is released.
+
+The diagram below shows the mode behaviour visually (for a [mode 2 transmitter](../getting_started/rc_transmitter_receiver.md#transmitters-for-aircraft)).
+
+<!-- The throttle will determine your airspeed, at 50% throttle the aircraft will hold its current altitude with a preset cruise air speed. You can descend and climb with pitch up and down command. In Altitude Mode you have full angle control over Pitch, Roll and Yaw. Once you center all the sticks you will have a level flying aircraft at the current altitude. If the wind blows the aircraft will drift in the direction of the wind. -->
 
 ![FW Altitude Mode](../../images/flight_modes/altitude_FW.png)
 
 
+
 ### Stabilized Mode {#stabilized_fw}
 
 [<img src="../../assets/site/difficulty_3.svg" title="Difficulty (Medium)" width="30px" />](#key_difficulty)&nbsp;[<img src="../../assets/site/remote_control.svg" title="Manual/Remote control required" width="30px" />](#key_manual)&nbsp;
 
-In Stabilized flight your plane will maintain a level flight. It will fight against the wind gusts automatically and maintain a horizontal posture. You will have a full manual angle control over the craft. You will be able to glide with your airplane when you lower the Throttle to 0% and the motor stops. In order to perform a turn you have to hold the command throughout the maneuver because if you release the roll the plane will stop turning and level itself. It is the same for the Pitch and Yaw commands as well.
+*Stabilized mode* allows you to climb/descend using pitch input and perform a coordinated turn if the roll/pitch sticks are non-zero. Releasing/centering the sticks puts the vehicle into straight and level flight (subject to wind - the vehicle heading and altitude are not maintained).
+
+> **Tip** *Stabilized mode* is much easier to fly than [Manual mode](#manual_fw) because the roll is angle controlled (you can't roll upside down) and it easy to level the vehicle by centering the control sticks.
+
+<!-- It will fight against the wind gusts automatically and maintain a horizontal posture. -->
```

Comment:
Can you confirm this (now commented text) is partially incorrect. In FW stabilized mode the vehicle will maintain altitude against wind when sticks are centred, but not heading.

Reply:
It will only maintain the attitude (roll/pitch), but not heading and not altitude.